Changelog.md

```diff
@@ -7,7 +7,9 @@ Language Features:
 Compiler Features:
  * Commandline Interface: Return exit code ``2`` on uncaught exceptions.
  * Commandline Interface: Add `--no-cbor-metadata` that skips CBOR metadata from getting appended at the end of the bytecode.
- * EVM: Basic support for the EVM version "Paris".
+ * EVM: Support for the EVM version "Paris".
+ * EVM: Introduce ``block.prevrandao`` in Solidity and ``prevrandao()`` in inline assembly for EVM versions >= paris.
+ * EVM: Deprecate ``block.difficulty`` and disallow ``difficulty()`` in inline assembly for EVM versions >= paris. The change is due to the renaming introduced by [EIP-4399](https://eips.ethereum.org/EIPS/eip-4399).
```

Comment:
Technically, that'd be

```suggestion
 * EVM: Deprecate ``block.difficulty`` and disallow ``difficulty()`` in inline assembly for EVM versions >= paris. The change is due to the renaming introduced by [EIP-4399](https://eips.ethereum.org/EIPS/eip-4399).
 * EVM: Introduce ``block.prevrandao`` in Solidity and ``prevrandao()`` in inline assembly for EVM versions >= paris.
 * EVM: Support for the EVM version "Paris".
```

;-). Not there isn't other unsorted ones right above, but it's always better to avoid having to resort this before the release, since that's just annoying on ``git blame``s and such.

Reply:
hahaha I confess I let the `Support...` part before the others for a better context of the changes. But ok, I will respect the alphabetic order also for the text after the category. ;)